content/en/1.13.x/concepts/updating-bottlerocket.markdown

```diff
@@ -0,0 +1,299 @@
+# Updating Bottlerocket
+
+This document covers the different ways to update Bottlerocket (for [ECS](#ecs), [Kubernetes](#kubernetes), and [SSM](#ssm)), as well as how to [lock your nodes to a specific release](#locking-to-a-specific-release).
+
+## Introduction
+
+Bottlerocket is designed to be updated in an [image-based fashion](https://github.com/bottlerocket-os/bottlerocket#updates).
+This means that updates are applied by replacing the entire image on disk, rather than applying a series of individual package updates.
+This is a departure from the traditional package-based Linux update model such as `apt` or `yum`, which are what you would find in Ubuntu or Amazon Linux.
+
+## Ways To Update
+
+There are a few ways to update Bottlerocket.
+The method you choose depends on the [Bottlerocket variant](https://github.com/bottlerocket-os/bottlerocket#variants) you are using and your environment.
+
+### ECS
+
+For the `aws-ecs-*` variants of Bottlerocket, we provide an [ECS updater](https://github.com/bottlerocket-os/bottlerocket-ecs-updater#how-it-works).
+Please see the [ECS updater installation documentation](https://github.com/bottlerocket-os/bottlerocket-ecs-updater#installation) for more information on how to install and use the ECS updater.
+
+Essentially, you will need to do the following:
```

Comment:
I am concerned about maintaining instructions in multiple places. (Like, very concerned). I think if the guidance is to look elsewhere for the instructions then we shouldn't summarize the instructions here. Also @stockholmux I'm concerned about having two sets of documentation to maintain for the Bottlerocket project-proper. Everything that's going here is already in Bottlerocket. How can we de-duplicate?

Reply:
@webern I more/less agree withy your point regarding summarizing non-Bottlerocket specific content. Theses docs should give enough context to enable someone to use other docs elsewhere.

As far as maintaining docs in multiple places, the plan is to bifurcate docs: user-facing docs (i.e. for those using Bottlerocket) are on the docs site; development information (i.e. for those contributing to Bottlerocket) are on bottlerocket-os/bottlerocket. 

There will likely be an overlap time as we build out this site to viability but at that point we can remove any information on bottlerocket-os/bottlerocket that exists over here.